go/ql/lib/semmle/go/frameworks/SQL.qll

```diff
@@ -81,18 +81,29 @@ module SQL {
                   "github.com/lann/squirrel"
                 ], "")
           |
-            // first argument to `squirrel.Expr`
-            fn.hasQualifiedName(sq, "Expr")
+            fn.hasQualifiedName(sq, ["Delete", "Expr", "Insert", "Select", "Update"])
             or
-            // first argument to the `Prefix`, `Suffix` or `Where` method of one of the `*Builder` classes
-            exists(string builder | builder.matches("%Builder") |
-              fn.(Method).hasQualifiedName(sq, builder, "Prefix") or
-              fn.(Method).hasQualifiedName(sq, builder, "Suffix") or
-              fn.(Method).hasQualifiedName(sq, builder, "Where")
+            // first argument to the `OrderBy`, `Prefix`, `Suffix` or `Where` method of one of the `*Builder` classes
+            exists(Method m, string builder | m = fn |
+              builder.matches("%Builder") and
```

Comment:
While you're editing this code, can you just explicitly list the types? It makes it much easier to search for and there aren't that many.

Reply:
```suggestion
              builder = ["DeleteBuilder", "InsertBuilder", "SelectBuilder", "UpdateBuilder"] and
```

Reply:
Sure, done here: e78e71c.